README.md

```diff
@@ -1,9 +1,25 @@
 # NodeJs backend server for Shoov.
 
-> This server is responsible for creating PRs for visual regressions, and executing live monitor tests.
+This server is responsible for creating PRs for visual regressions, and executing live monitor tests.
 
-## Installation
+## Install
 
-Install: `npm install`
+Install all node dependencies:
 
-Execute: `BACKEND_URL=https://backend.com pm2 start bin/www --name=node-server`
+    $ npm install
+
+## Execute
+
+    $ BACKEND_URL=http://192.168.1.32/shoov.local \
+      LOGGLY_TOKEN=802390n3-c3d4-4d44-a022-0nv7a34n8934 \
+      DEBUG=true \
+      pm2 start bin/www --name=node-server
+
+Possible environments:
+
+* __BACKEND_URL__ - The url address of backend server. It's can't be alias for your ip (for example shoov.local) because docker doesn't know about your hosts. 
```

Comment:
`The url address of backend server. When used locally it be the IP address and not the site alias.`